mailbox/event/json/src/test/java/org/apache/james/event/json/AddedSerializationTest.java

```diff
@@ -88,7 +89,8 @@
         "          \"userFlags\":[\"User Custom Flag\"]}," +
         "        \"size\": 45,  " +
         "        \"internalDate\": \"2018-12-14T09:41:51.541Z\"," +
-        "        \"messageId\": \"42\"" +
+        "        \"messageId\": \"42\"," +
+        "        \"threadId\":\"42\""   +
```

Comment:
This is a breaking change. We need to handle the old format.

Please add the test.